src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const ONE_DAY_RENT_COST = 40;
+  const THREE_DAYS_RENT_DISCOUNT = 20;
+  const SEVEN_DAYS_RENT_DISCOUNT = 50;
+  const SMALL_TERM_RENT = 3;
+  const LONG_TERM_RENT = 7;
+  const TOTAL_RENT = ONE_DAY_RENT_COST * days;
+
+  if (days < SMALL_TERM_RENT) {
+    return TOTAL_RENT;
+  }
+
+  if (days >= LONG_TERM_RENT) {
+    return TOTAL_RENT - SEVEN_DAYS_RENT_DISCOUNT;
+  }
+
+  return TOTAL_RENT - THREE_DAYS_RENT_DISCOUNT;
```

Comment:
Better to change logic in `if`s to make `return TOTAL_RENT;` as a default return(the last one)
And `TOTAL_RENT - THREE_DAYS_RENT_DISCOUNT` should be in `if` block